### PR TITLE
Feature/weth 2 eth

### PR DESCRIPTION
**Pull request by W3stside**
WIP *edit

Balance showed as `web3.getBalance(account)`  aka normal ETH

1. inserted W-ETH in `TokenCode` as typing
2. changed `getCurrentBalance` function to check for correct ETH

**edit -> will need auto conversion to W-ETH when selling / buying

### CODE REVIEW

File: src/reducers/tokenBalances.ts [hunk 1]
```diff
@@ -7,16 +7,16 @@ import { setTokenBalance } from 'actions/tokenBalances'
 // const zeroBalance = codeList.reduce((acc, code) => (acc[code] = 0, acc), {}) as TokenBalances
 
 // TODO: fill in when we have actions + fill in rest of Tokens + proper typing
-export default handleActions<TokenBalances | any>(
+export default handleActions<TokenBalances>(
   {
     [setTokenBalance.toString()]: (state, action) => ({
       ...state,
-      [action.payload.tokenName]: action.payload.balance,
+      [action.payload['tokenName']]: action.payload['balance'],
```
Comment (Velenir) [inline src/reducers/tokenBalances.ts:14]: why? TS fault?

Comment (W3stside) [reply src/reducers/tokenBalances.ts:14]: yeah weird error saying `tokenName` non existant on `TokenBalances` type

File: src/api/index.ts [hunk 2]
```diff
@@ -31,14 +31,21 @@ export const getETHBalance = async (account?: Account) => {
...
+  }
 
+  const { Tokens } = await promisedAPI
   // account would normally be taken from redux state and passed inside an action
   // but just in case
-  account = await fillDefaultAccount(account)
 
-  return Tokens.getTokenBalance('ETH', account)
+  // should probably change name here to WETH
+  return Tokens.getTokenBalance('W-ETH', account)
```
Comment (Velenir) [inline src/api/index.ts:48]: what about using `tokenName` here, maybe defaulting it to `W-ETH`

Comment (W3stside) [reply src/api/index.ts:48]: that was my initial idea, and it worked but I'm honestly having second thoughts about this W-ETH thing. Rn I'm on the same PR and reverting back a commit. I think I want to keep the ETH name the same and just run a conditional there to check balance different if `tokenName` === `ETH`

File: src/reducers/tokenBalances.ts (hunk 1 above)
Comment (Velenir) [reply src/reducers/tokenBalances.ts:14]: change to 
```
handleActions<TokenBalances, {tokenName: string, balance: string}>(
```

second temlate arg is payload

**Comment by W3stside** [comment]
Prefer #201 over this

**Comment by W3stside** [comment]
closing in favour of #201

**Pull request closed without merging**